Documentation/README.md

```diff
@@ -11,3 +11,4 @@ The following .NET Framework documentation is available from Microsoft:
 
 - [.NET Framework Documentation](https://docs.microsoft.com/dotnet/articles/framework/)
 - [.NET Core Documentation](https://docs.microsoft.com/dotnet/articles/core/)
+- [.NET Standard and General .NET Development](https://docs.microsoft.com/dotnet/standard/)
```

Comment:
Hi,
Wouldn't be more accurate this [link with regards to .NET standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)?

Reply:
That's a good point, @sebainones. I think it's best to include two links: the one that I included for general .NET development, and the one for .NET Standard that you suggested.